src/pages/Notifications/List/BundlePage.tsx

```diff
@@ -45,29 +49,85 @@ export const NotificationListBundlePage: React.FunctionComponent<NotificationLis
+
+    const getInitialApplications = useMemo(
+        () => {
+            if (getApplications.payload) {
+                return getApplications.payload.value as Facet[];
+            } else {
+                return [];
+            }
+        }, [ getApplications.payload ]);
+
+    if (notificationsOverhaul) {
+
+        const handleTabClick = (event, tabIndex) => {
+            setActiveTabKey(tabIndex);
+        };
+
+        return (
+            <><PageHeader
+                title={ pageTitle() }
+                subtitle={ <span>This service allows you to configure which notifications different
+                    users within your organization will be entitled to receiving. To do this, create behavior groups and apply
+                <a href={ localUrl(`/user-preferences/notifications/${props.bundle.name}`,
+                    getInsights().chrome.isBeta()) }> User Preferences</a>.</span> }
+                action={ eventLogButton() }
+            />
+            <Flex direction={ { default: 'column' } }>
+                <FlexItem>
+                    <Tabs activeKey={ activeTabKey } onSelect={ handleTabClick }>
+                        <Tab eventKey={ 0 } title={ <TabTitleText>Red Hat Enterprise Linux</TabTitleText> }>
+                            <Main><BundlePageBehaviorGroupContent applications={ getInitialApplications } bundle={ props.bundleTabs[0] } /></Main>
+                        </Tab>
+                        <Tab eventKey={ 1 } title={ <TabTitleText>Console</TabTitleText> }>
+                            <Main><BundlePageBehaviorGroupContent applications={ getInitialApplications } bundle={ props.bundleTabs[1] } /></Main>
+                        </Tab>
+                        <Tab eventKey={ 2 } title={ <TabTitleText>Openshift</TabTitleText> }>
+                            <Main><BundlePageBehaviorGroupContent applications={ getInitialApplications } bundle={ props.bundleTabs[2] } /></Main>
+                        </Tab>
+                    </Tabs>
+                </FlexItem>
+            </Flex>
+            </>
+        );
+    } else {
+        return (
+            <><PageHeader
+                title={ `${Messages.pages.notifications.list.title} | ${props.bundle.displayName}` }
+                subtitle={ <span>This service allows you to configure which notifications different
+                    users within your organization will be entitled to receiving. To do this, create behavior groups and apply
+                    them to different events. Users will be able to opt-in or out of receiving authorized event notifications in their
+                <a href={ localUrl(`/user-preferences/notifications/${props.bundle.name}`,
+                    getInsights().chrome.isBeta()) }> User Preferences</a>.</span> }
+                action={ eventLogButton() }
+            />
+
+            <TabComponent configuration={ props.children } settings={ props.children }>
```

Comment:
where did this section go? I don't see the time config in the mocks but curious if that was on purpose or not.

Reply:
I think this is supposed to go to a different page. I don't remember where though. We might need a story for that.

Reply:
yeah, worth reaching out to UXD.

Reply:
I agree. Will do.